flopy/modpath/mp6.py

```diff
@@ -43,14 +44,16 @@ class Modpath6(BaseModel):
         The name of the executable to use.
     modflowmodel : flopy.modflow.Modflow
         MODFLOW model object with one of LPF, BCF6, or UPW packages.
-    dis_file : str or PathLike
-        Required dis file name.
+    dis_file : str or PathLike, optional
+        dis file name. If not provided, this is obtained from modflowmodel.
     dis_unit : int, default 87
         Optional dis file unit number.
-    head_file : str or PathLike
-        Required filename of the MODFLOW output head file.
-    budget_file : str or PathLike
-        Required filename of the MODFLOW output cell-by-cell budget file.
+    head_file : str or PathLike, optional
+        Filename of the MODFLOW output head file.
+        If not provided, this is obtained from modflowmodel.
+    budget_file : str or PathLike, optional
+        Filename of the MODFLOW output cell-by-cell budget file.
+        If not provided, this is obtained from modflowmodel.
```

Comment:
These were previously both "Required" and "Optional". Corrected to use information from modflowmodel if not provided (is None).